src/sentry/static/sentry/app/redux/actions/substance.js

```diff
@@ -0,0 +1,102 @@
+import axios from 'axios';
+
+export const SUBSTANCES_GET_REQUEST = 'SUBSTANCES_GET_REQUEST';
+export const SUBSTANCES_GET_SUCCESS = 'SUBSTANCES_GET_SUCCESS';
+export const SUBSTANCES_GET_FAILURE = 'SUBSTANCES_GET_FAILURE';
+export const SUBSTANCES_TOGGLE_SELECT_ALL = 'SUBSTANCES_TOGGLE_SELECT_ALL';
```

Comment:
Do we also need an action to deselect all?

Reply:
This action handles both, as there is a flag in the action called `doSelect` which specifies if it should be off or on.